### PR TITLE
Update Sats to USD conversion rate

### PR DESCRIPTION
**Pull request by johnpaulkiser**
$3 -> $10.
Although the current rate is just over $11, due to bitcoin's unforeseeable price fluctuations I believe $10 to be a sightly estimate for the docs.

### CODE REVIEW

**Comment by johnpaulkiser** [comment]
I have no problem with that. Although, I do think it's important to stress the greater purchasing power inherent in BSQ against BTC which a conversion rate to me seems the easiest to understand.

File: user-dao-intro.adoc [hunk 1]
```diff
@@ -71,9 +71,9 @@ In the Bisq DAO, trading fees are collected from traders and distributed to cont
 [NOTE]
 .Where does new BSQ actually come from?
 ====
-Recall that a BSQ token is merely colored bitcoin. When a contributor makes their compensation request in the DAO, they must also include the tiny amount of bitcoin to be 'colored' as BSQ (<<dao-technical-overview#,the spec>> requires 100 satoshis). So if a contributor requests 1,000 BSQ, they will need to include 100 * 1,000 = 100,000 satoshis with their compensation request—just about 3 USD at current rates.
+Recall that a BSQ token is merely colored bitcoin. When a contributor makes their compensation request in the DAO, they must also include the tiny amount of bitcoin to be 'colored' as BSQ (<<dao-technical-overview#,the spec>> requires 100 satoshis). So if a contributor requests 1,000 BSQ, they will need to include 100 * 1,000 = 100,000 satoshis with their compensation request—just about 10 USD at current rates.
```
Comment (m52go) [inline user-dao-intro.adoc]:
```suggestion
Recall that a BSQ token is merely colored bitcoin. When a contributor makes their compensation request in the DAO, they must also include the tiny amount of bitcoin to be 'colored' as BSQ (<<dao-technical-overview#,the spec>> requires 100 satoshis). So if a contributor requests 1,000 BSQ, they will need to include 100 * 1,000 = 100,000 satoshis with their compensation request—just about 10 USD at a rate of 10,000 USD/BTC.
```

**Comment by m52go** [comment]
@johnpaulkiser any thoughts?

**Comment by johnpaulkiser** [comment]
I think that makes it much more clear. I say we merge this and revisit it again during a time of price instability.

**Comment by m52go** [comment]
utACK

**Pull request merged**